test/search/test_organized_index.cpp

```diff
@@ -132,16 +135,10 @@ TEST (PCL, Organized_Neighbor_Search_Pointcloud_Nearest_K_Neighbour_Search)
 
     std::priority_queue<prioPointQueueEntry> pointCandidates;
 
-
-    // push all points and their distance to the search point into a priority queue - bruteforce approach.
-    for (std::size_t i = 0; i < cloudIn->points.size (); i++)
+    for (auto it = cloudIn->begin(); it != cloudIn->end(); ++it)
```

Comment:
If the loop doesn't need, don't use non-const iterators. Prefer `cbegin` and `cend` where possible

Reply:
Do pointcloud iterators support `cbegin` and `cend`?
Could not find it here:
https://github.com/PointCloudLibrary/pcl/blob/8bcde6f06c94d401fc33d695eb8eac15194538c1/common/include/pcl/point_cloud.h#L434-L440

Reply:
If no, `cloudIn->points.cbegin()` and create a issue in the meanwhile